src/spyglass/spikesorting/v0/spikesorting_sorting.py

```diff
@@ -127,6 +127,8 @@ class SpikeSorting(SpyglassMixin, dj.Computed):
     time_of_sort: int   # in Unix time, to the nearest second
     """
 
+    _parallel_make = True
```

Comment:
Good for now

Outdated approach per [this PR](https://github.com/datajoint/datajoint-python/pull/1204), but we're waiting on datajoint 0.14.4 release to be able to implement